src/content-tags-drawer/ContentTagsCollapsible.jsx

```diff
@@ -117,10 +119,31 @@ const ContentTagsCollapsible = ({ contentId, taxonomyAndTagsData, editable }) =>
   const [isOpen, open, close] = useToggle(false);
   const [addTagsButtonRef, setAddTagsButtonRef] = React.useState(null);
 
+  const [searchTerm, setSearchTerm] = React.useState(null);
```

Comment:
```suggestion
  const [searchTerm, setSearchTerm] = React.useState(/** @type {string|null} */ (null));
```
(if you want to keep `null` as the "no search term" value. But I'd say that using `""` as the "no search term" value is even simpler.)

Also, if you put `// @ts-check` at the top of this file (which you should), you'll see there are a few minor errors that should be easily fixed. Mostly that `useCallback` requires a second `[]` parameter.

Reply:
Agreed I think the "" is cleaner, so I switched it to that, I also fixed the other typing issues after adding the `// @ts-check`